components/Footer.js

```diff
@@ -14,6 +14,10 @@ const Footer = () => {
           name: "Contributors",
           link: "/contributors",
         },
+        {
+          name: "Partners",
+          path: "/partners",
```

Comment:
This should be 
```suggestion
          link: "/partners",
```
 as all the other objects are with the field name `link`.